components/arm/arm.go

```diff
@@ -72,17 +72,55 @@ type Arm interface {
 	referenceframe.InputEnabled
 
 	// EndPosition returns the current position of the arm.
+	//
+	//    myArm, err := arm.FromRobot(machine, "my_arm")
```

Comment:
You seem to do `if err != nil { logger.Error(err); return }` logic occasionally in this file and sometimes forgo it. Either way seems fine to me (I lean toward not showing how to handle the error in this case and trusting that the user will know to handle all `err` values), but probably want to pick one.